[2023-09-13T07:38:28.672+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-19T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.709+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-19T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.710+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:28.781+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-19 00:00:00+00:00
[2023-09-13T07:38:28.807+0000] {standard_task_runner.py:57} INFO - Started process 95 to run task
[2023-09-13T07:38:28.839+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-19T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpire2kvt4']
[2023-09-13T07:38:28.842+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask extract_data
[2023-09-13T07:38:29.089+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-19T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:29.347+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-19T00:00:00+00:00'
[2023-09-13T07:38:29.625+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:29.650+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230819T000000, start_date=20230913T073828, end_date=20230913T073829
[2023-09-13T07:38:29.702+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:29.722+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:07.778+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-19T00:00:00+00:00 [queued]>
[2023-09-15T11:31:07.829+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-19T00:00:00+00:00 [queued]>
[2023-09-15T11:31:07.842+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:07.897+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-19 00:00:00+00:00
[2023-09-15T11:31:07.943+0000] {standard_task_runner.py:57} INFO - Started process 328 to run task
[2023-09-15T11:31:07.960+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-19T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpzs1pxlf3']
[2023-09-15T11:31:07.965+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask extract_data
[2023-09-15T11:31:08.146+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-19T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:08.483+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-19T00:00:00+00:00'
[2023-09-15T11:31:08.794+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:08.831+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230819T000000, start_date=20230915T113107, end_date=20230915T113108
[2023-09-15T11:31:08.885+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:08.915+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
